### PR TITLE
Emily's branch!! :)

### PR DESCRIPTION
**Pull request by jjfeng**

### CODE REVIEW

File: easier_net/easier_net.py [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+import sier_net #technically should be from sier_net import SierNetEstimator
+
+class EasierNetEstimator():
+    """ This is the Ensembling by Averaging Sparse-Input Hierarchical Networks(EASIER-NET)"""
+    #i think my init needs a seed
+    def __init__(
+        self, #num inits >=5
+        # seed=12,
+        # data_file="_output/data.npz",
+        num_classes=0,
```
Comment (jjfeng) [inline easier_net/easier_net.py:15]: at some point, we should make an easier net for regression and one for classification. this is fine for now though

File: easier_net/easier_net.py [hunk 2]
```diff
@@ -0,0 +1,186 @@
+from ensurepip import bootstrap
+# from imblearn import RandomUnderSampler
+import numpy as np
+
+import pickle
+import sier_net #technically should be from sier_net import SierNetEstimator
+
+class EasierNetEstimator():
```
Comment (jjfeng) [inline easier_net/easier_net.py:8]: i would add comments to explain what each parameter does.

File: easier_net/easier_net.py [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+        # seed=12,
+        # data_file="_output/data.npz",
+        num_classes=0,
+        input_filter_layer=True,
+        n_layers=2,
+        n_hidden=10,
+        num_inits=5,       #change this to n_estimators
+        max_iters=40,
+        max_prox_iters=0,
+        bootstrap=False,
```
Comment (jjfeng) [inline easier_net/easier_net.py:22]: i dont think this is used, so remove

**Review by jjfeng — COMMENTED** [review]
I just peeked at this code, but noticed a lot of old code and some stuff that is TBD. Please remove as much old code as much as possible and make the file as clean as possible for me to do a formal review.

File: easier_net/easier_net.py (hunk 3 above)
Comment (emumily) [reply easier_net/easier_net.py:22]: I think we need n_estimators for creating the size of the ensemble? Removing the other 3 arguments.

**Comment by emumily** [comment]
closing PR and creating a new one once code has been cleaned

**Pull request closed without merging**